source/_integrations/iotty.markdown

```diff
@@ -34,15 +34,23 @@ This integration currently supports the following iotty devices:
   - [2-Switch Controller](https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925)
   - [3-Switch Controller](https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269)
   - [4-Switch Controller](https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901)
-  EU version:
+  EU version: \
   - [iotty Smart Switch](https://iotty.uk/collections/frontpage/products/iotty-smart-switch)
   - [iotty Smart Switch (variant)](https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082)
   - [iotty Plus Interruttore Intelligente](https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli)
 
+- iotty Shutter \
+  - [iotty Smart Shades Switch](https://iotty.uk/collections/frontpage/products/e2s-plus-smart-shades-switch-for-shutters-and-blinds)
+  - [iotty Plus Interruttore Intelligente per Tende e Tapparelle](https://iotty.it/collections/prodotti-singoli/products/i3s-plus-interruttore-intelligente-per-tende-e-tapparelle) (currently only available for the Italian market)
+
 ## Supported entities
 
 Each iotty device gets mapped into one Home Assistant device per gang, each with its own Switch entity.
 
 ### Switches
 
 The main, unnamed, switch entity controls the light switch, turning it on or off the gang related to it.
+
+### Covers
```

Comment:
please also add cover to the header, under ha_category and platform. It is too far out for me to be able to add a suggestion there

Reply:
I'm not sure I understand what you mean here. I already have a Cover section here, at the end of the file.

Reply:
apologies, should have been more specific. I meant the file metadata, under line 6 and 13

Reply:
Done